traveler-ios-ui/TravelerKitUI/ViewControllers/OrderDetailViewController.swift

```diff
@@ -149,3 +154,23 @@ extension OrderDetailViewController: CancellationQuoteFetchDelegate {
         present(alert, animated: true, completion: nil)
     }
 }
+
+extension OrderDetailViewController: EmailTicketsDelegate {
+    func emailDidSucceed() {
```

Comment:
Feedback from TribalScale:

They mentioned it would be nice (in other delegates) if the delegate passed back original query objects. We will make a plan to change our delegates to match this, but I think moving forward its good to include this practice.

In this case the query object would be the `Order` itself

i.e.

```

```